roxie/udplib/udpsim.cpp

```diff
@@ -154,6 +157,24 @@ void initOptions(int argc, const char **argv)
     udpTraceFlow = options->getPropBool("@udpTraceFlow", false);
     packetsPerThread = options->getPropInt("@packetsPerThread");
     udpTestUseUdpSockets = !options->getPropBool("@useQueue");
+    udpTestSocketDelay = options->getPropInt("@udpTestSocketDelay", 0);
+    udpTestSocketJitter = options->getPropBool("@udpTestSocketJitter", 0);
+    udpTestVariableDelay = options->getPropBool("@udpTestVariableDelay");
+    if (udpTestSocketDelay && udpTestUseUdpSockets)
```

Comment:
bug: This test should go after the other tests that could set udpTestSocketDelay

Reply:
Moved